snooty.toml

```
@@ -28,5 +29,5 @@ mdb-server = "MongoDB Server"
 stable-api = "Stable API"
 api = "https://mongodb.github.io/mongo-java-driver/{+version-number+}/apidocs/mongodb-driver-kotlin-sync"
 java-api = "https://mongodb.github.io/mongo-java-driver/{+version-number+}"
-core-api = "{+java-api+}/apidocs/mongodb-driver-core/"
 kotlin-docs = "https://kotlinlang.org"
+core-api = "https://mongodb.github.io/mongo-java-driver/{+version-number+}/apidocs/mongodb-driver-core/com/mongodb"
```

This change will break existing links that appear in the Kotlin Sync repo. You should use the following value for `core-api` and update any links you have to make sure they work. You can probably just duplicate the [`snooty.toml` file from the current main branch of the repo](https://github.com/mongodb/docs-kotlin-sync/blob/master/snooty.toml)

```suggestion
core-api = "{+java-api+}/apidocs/mongodb-driver-core/"
```

---

Ahh thanks for catching this! Updated